### ISSUE TITLE
Update SeedManager to use bignumber.js

### ISSUE THREAD

**Issue by BeepBot99** [Mesure73L/mesure.x10.mx#173]
### Issue Contents

I just created `SeedManager` in PHP. It will be at the end of this issue.
The problem is, it was giving slightly different answers than the JavaScript SeedManager was. After further investigation, I realized that the JavaScript was rounding the big numbers to the nearest thousand, while the PHP was not. Therefore, I want to implement [bignumber.js](https://mikemcl.github.io/bignumber.js/) into the JavaScript so that it stores the numbers as strings instead of numbers, so it will not round them. I am asking for permission to implement that into SeedManager in the #92 branch.
In case you may want it, here is the PHP for `SeedManager`:
```php
<?php
$state = 1000000000; # Starting seed

function nextFloat() {
	global $state;
	$m = 0x80000000;
	$a = 1103515245;
	$c = 12345;
	$state = ($a * $state + $c) % $m;
	return $state / ($m - 1);
}
echo nextFloat(); # nextFloat() gives the next random number. The first time you call it should give the same number as the first item in the array for the JavaScript SeedManager.
```

**Comment by BeepBot99** [comment]
# TL;DR
The author created a PHP version of SeedManager, but it was found to produce slightly different results than the JavaScript version. The problem was identified to be the JavaScript version rounding the big numbers to the nearest thousand, while the PHP version was not. The author strongly recommends implementing `bignumber.js` into the JavaScript version to prevent the rounding of numbers and ensure consistency in the generated results.

**Comment by Mesure73L** [comment]
You are allowed to use the #92 branch until tomorrow.

**Comment by Mesure73L** [comment]
> The author created a PHP version of SeedManager, but it was found to produce slightly different results than the JavaScript version. The problem was identified to be the JavaScript version rounding the big numbers to the nearest thousand, while the PHP version was not. The author strongly recommends implementing `bignumber.js` into the JavaScript version to prevent the rounding of numbers and ensure consistency in the generated results.

bro that was chatgpt edited, only chatgpt would say "The author" and not "I"

**Comment by BeepBot99** [comment]
@Mesure73L Can I do this in branch #92, or should I create a new branch from it?

**Comment by Mesure73L** [comment]
> To help my writing process, I used these Grammarly AI prompts:
> **Prompts created by Grammarly**
> 
> * "Write a project progress report"

yeah ik thats ai, im not reading it

**Comment by Mesure73L** [comment]
@BeepBot99 please actually do this instead of getting grammarly ai to write you a message about doing it

**Comment by BeepBot99** [comment]
I'm literally doing it right now.

**Comment by BeepBot99** [comment]
My console right now:
![image](https://github.com/Mesure73L/mesure.x10.mx/assets/74377508/a0c39e12-7587-4162-8def-6b437d9ba55d)

**Comment by BeepBot99** [comment]
@Mesure73L Can you please add a `<script>` tag for the `not-an-api/libraries/bignumber/bignumber.js` file in all of the challenge html files?

**Comment by Mesure73L** [comment]
> @Mesure73L Can you please add a `<script>` tag for the `not-an-api/libraries/bignumber/bignumber.js` file in all of the challenge html files?

Do you still need me to do this?

**Comment by BeepBot99** [comment]
Yes.

**Comment by BeepBot99** [comment]
Make sure it is in the head.

**Comment by Mesure73L** [comment]
Why does it need to be in the head? Should I put it in the head with `defer` or not?

**Comment by BeepBot99** [comment]
No, it is a library. No `defer` because it does not need to run any code immediately, only when the JavaScript calls it.

**Comment by Mesure73L** [comment]
Done.

**Comment by BeepBot99** [comment]
Great, thank you!

**Comment by Mesure73L** [comment]
You're welcome!
Thank you for saying thank you!

**Comment by BeepBot99** [comment]
Thank you for saying you're welcome!
Thank you for saying thank you for saying thank you!
You're welcome for saying thank you!

**Comment by Mesure73L** [comment]
Thank you for saying thank you for saying you're welcome!
Thank you for saying thank you for saying thank you for saying thank you!
You're welcome for saying you're welcome!
You're welcome for saying thank you for saying thank you for saying thank you!
Thank you for saying you're welcome for saying thank you!

**Comment by BeepBot99** [comment]
Thank you for saying thank you for saying thank you for saying you're welcome!
Thank you for saying thank you for saying thank you for saying thank you for saying thank you!
Thank you for saying you're welcome for saying you're welcome!
Thank you for saying you're welcome for saying thank you for saying thank you for saying thank you!
Thank you for saying thank you for saying you're welcome for saying thank you!
You're welcome for saying thank you for saying you're welcome!
You're welcome for saying thank you for saying thank you for saying thank you!
You're welcome for saying you're welcome for saying thank you!

**Comment by Mesure73L** [comment]
ok im done with this

**Comment by BeepBot99** [comment]
NOOOOOOOO.
